server/lib/tuist_web/marketing/live/marketing_case_studies_live.html.heex

```diff
@@ -0,0 +1,98 @@
+<TuistWeb.Marketing.Layouts.marketing {assigns}>
+  <main
+    id="marketing-case-studies"
+    phx-hook="ScrollToTop"
+    data-scroll-target='[data-part="filters-bar"]'
+  >
+    <TuistWeb.Marketing.MarketingComponents.navbar current_user={@current_user} />
+
+    <TuistWeb.Marketing.MarketingComponents.header_background />
+
+    <header data-part="header">
+      <img data-part="logo" src={~p"/marketing/images/case-studies/logo.webp"} alt="" />
+      <h1 data-part="title">
+        {dgettext("marketing", "Case Studies,")}
+      </h1>
+      <p data-part="description">
+        {dgettext(
+          "marketing",
+          "Discover how teams around the world use Tuist to build high-quality apps faster, streamline workflows, and save countless hours of engineering time."
```

Comment:
I'd suggest applying a more aggressive `max-width`, right now, the text is in too long of a line, which makes it hard to read (and is not in line with the design):
<img width="1982" height="908" alt="image" src="https://github.com/user-attachments/assets/7d2f61d9-293f-454e-b260-e63737f29e46" />